<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>499</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1247</width>
    <height>499</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1247</width>
    <height>499</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pascal_VOC_XML to Faster RCNN format Converter </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bolder;
font-size: 15px;
color: #007FFF;</string>
    </property>
    <property name="text">
     <string>XML path</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bolder;
font-size: 15px;
color: #007FFF;</string>
    </property>
    <property name="text">
     <string>Desired output csv file name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xml_path_txt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #00aaaa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_csv_path_txt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #00aaaa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bolder;
font-size: 15px;
color: #007FFF;</string>
    </property>
    <property name="text">
     <string>Desired output annotation file name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_annoation_path_txt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>221</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #00aaaa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_btn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black;
font-size: 50px;
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Pascal_VOC_XML to Faster RCNN format Converter </string>
    </property>
   </widget>
   <widget class="QLabel" name="result_prompt">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>400</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>865</x>
      <y>190</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>220</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;
font-size: 8px</string>
    </property>
    <property name="text">
     <string>browse the main folder in which 
 images and annotations are placed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;
font-size: 8px</string>
    </property>
    <property name="text">
     <string>e.g sample.csv</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
